notes.md

```diff
@@ -0,0 +1,215 @@
+# Why does `:` only sometimes work?
+I don't like `:` being listed as a `source_column_selector` when it often errors.
+
+`select(df, :)`
+: works
+
+`transform(df, Cols(:) => ByRow(max))`
+: works
+
+`transform(df, : => ByRow(max))` errors
+: ERROR: syntax: whitespace not allowed after ":" used for quoting
+
+Similarly, regular expressions also sometimes must be wrapped in `Cols`.
```

Comment:
This is also a limitation of the fact that regular expressions are defined in Base Julia. We cannot adjust their behavior. Therefore we have `Cols` wrapper that can be put around anything.

Reply:
`Cols` around regex is only needed with broadcasting